citf_options/citf_options.go

```diff
@@ -1,52 +1,56 @@
 package citfoptions
 
-// CreateOptions specifies which fields of CITF should be included when created or reloadedd
+import "github.com/openebs/CITF/utils/log"
+
+// CreateOptions specifies which fields of CITF should be included when created or reloaded
```

Comment:
It looks like `BuilderPattern` should be used to build the CreateOptions struct. Rather than creating every possible permutation & combination for this struct, please evaluate if it should expose a Fluent API that can be used as per the requirement at the calling code.

Reply:
Sure @AmitKumarDas  I'll do that.